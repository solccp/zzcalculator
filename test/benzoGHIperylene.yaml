---
structure_id: 1
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 20]
- [19, 15]
- [17, 18]
- [16, 9]
- [10, 3]
- [4, 7]
- [5, 21]
- [8, 14]
- [13, 12]
- [11, 22]
---
structure_id: 2
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 20]
- [19, 15]
- [17, 18]
- [16, 9]
- [10, 3]
- [4, 5]
- [21, 22]
- [7, 11]
- [8, 14]
- [12, 13]
---
structure_id: 3
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 20]
- [19, 15]
- [17, 18]
- [16, 9]
- [10, 3]
- [4, 5]
- [21, 22]
- [7, 8]
- [11, 12]
- [13, 14]
---
structure_id: 4
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 20]
- [19, 15]
- [17, 18]
- [16, 9]
- [10, 3]
- [4, 5]
- [21, 22]
rings:
- [7, 8, 14, 13, 12, 11]
---
structure_id: 5
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 20]
- [19, 15]
- [17, 18]
- [16, 9]
- [10, 3]
- [8, 14]
- [12, 13]
rings:
- [4, 5, 21, 22, 11, 7]
---
structure_id: 6
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [20, 19]
- [4, 7]
- [5, 21]
- [22, 11]
- [12, 13]
- [14, 8]
- [9, 16]
- [10, 15]
- [17, 18]
---
structure_id: 7
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [20, 19]
- [4, 7]
- [5, 21]
- [22, 11]
- [12, 13]
- [14, 8]
- [9, 10]
- [15, 17]
- [16, 18]
---
structure_id: 8
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [20, 19]
- [4, 7]
- [5, 21]
- [22, 11]
- [12, 13]
- [14, 8]
rings:
- [9, 10, 15, 17, 18, 16]
---
structure_id: 9
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [20, 19]
- [4, 5]
- [21, 22]
- [7, 11]
- [12, 13]
- [14, 8]
- [9, 16]
- [10, 15]
- [17, 18]
---
structure_id: 10
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [20, 19]
- [4, 5]
- [21, 22]
- [7, 11]
- [12, 13]
- [14, 8]
- [9, 10]
- [15, 17]
- [16, 18]
---
structure_id: 11
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [20, 19]
- [4, 5]
- [21, 22]
- [7, 11]
- [12, 13]
- [14, 8]
rings:
- [9, 10, 15, 17, 18, 16]
---
structure_id: 12
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [20, 19]
- [4, 5]
- [21, 22]
- [7, 8]
- [11, 12]
- [13, 14]
- [9, 16]
- [10, 15]
- [17, 18]
---
structure_id: 13
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [20, 19]
- [4, 5]
- [21, 22]
- [7, 8]
- [11, 12]
- [13, 14]
- [9, 10]
- [15, 17]
- [16, 18]
---
structure_id: 14
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [20, 19]
- [4, 5]
- [21, 22]
- [7, 8]
- [11, 12]
- [13, 14]
rings:
- [9, 10, 15, 17, 18, 16]
---
structure_id: 15
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [20, 19]
- [4, 5]
- [21, 22]
- [9, 16]
- [10, 15]
- [17, 18]
rings:
- [7, 8, 14, 13, 12, 11]
---
structure_id: 16
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [20, 19]
- [4, 5]
- [21, 22]
- [9, 10]
- [15, 17]
- [16, 18]
rings:
- [7, 8, 14, 13, 12, 11]
---
structure_id: 17
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [20, 19]
- [4, 5]
- [21, 22]
rings:
- [7, 8, 14, 13, 12, 11]
- [9, 10, 15, 17, 18, 16]
---
structure_id: 18
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [20, 19]
- [12, 13]
- [14, 8]
- [9, 16]
- [10, 15]
- [17, 18]
rings:
- [4, 5, 21, 22, 11, 7]
---
structure_id: 19
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [20, 19]
- [12, 13]
- [14, 8]
- [9, 10]
- [15, 17]
- [16, 18]
rings:
- [4, 5, 21, 22, 11, 7]
---
structure_id: 20
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [20, 19]
- [12, 13]
- [14, 8]
rings:
- [4, 5, 21, 22, 11, 7]
- [9, 10, 15, 17, 18, 16]
---
structure_id: 21
number_of_atoms: 22
double_bonds:
- [1, 6]
- [17, 18]
- [16, 9]
- [4, 7]
- [5, 21]
- [8, 14]
- [13, 12]
- [11, 22]
rings:
- [2, 3, 10, 15, 19, 20]
---
structure_id: 22
number_of_atoms: 22
double_bonds:
- [1, 6]
- [17, 18]
- [16, 9]
- [4, 5]
- [21, 22]
- [7, 11]
- [8, 14]
- [12, 13]
rings:
- [2, 3, 10, 15, 19, 20]
---
structure_id: 23
number_of_atoms: 22
double_bonds:
- [1, 6]
- [17, 18]
- [16, 9]
- [4, 5]
- [21, 22]
- [7, 8]
- [11, 12]
- [13, 14]
rings:
- [2, 3, 10, 15, 19, 20]
---
structure_id: 24
number_of_atoms: 22
double_bonds:
- [1, 6]
- [17, 18]
- [16, 9]
- [4, 5]
- [21, 22]
rings:
- [2, 3, 10, 15, 19, 20]
- [7, 8, 14, 13, 12, 11]
---
structure_id: 25
number_of_atoms: 22
double_bonds:
- [1, 6]
- [17, 18]
- [16, 9]
- [8, 14]
- [12, 13]
rings:
- [2, 3, 10, 15, 19, 20]
- [4, 5, 21, 22, 11, 7]
---
structure_id: 26
number_of_atoms: 22
double_bonds:
- [1, 2]
- [6, 5]
- [20, 19]
- [21, 22]
- [3, 10]
- [4, 7]
- [11, 12]
- [13, 14]
- [8, 9]
- [15, 17]
- [16, 18]
---
structure_id: 27
number_of_atoms: 22
double_bonds:
- [1, 2]
- [6, 5]
- [20, 19]
- [21, 22]
- [3, 4]
- [7, 11]
- [12, 13]
- [14, 8]
- [9, 16]
- [10, 15]
- [17, 18]
---
structure_id: 28
number_of_atoms: 22
double_bonds:
- [1, 2]
- [6, 5]
- [20, 19]
- [21, 22]
- [3, 4]
- [7, 11]
- [12, 13]
- [14, 8]
- [9, 10]
- [15, 17]
- [16, 18]
---
structure_id: 29
number_of_atoms: 22
double_bonds:
- [1, 2]
- [6, 5]
- [20, 19]
- [21, 22]
- [3, 4]
- [7, 11]
- [12, 13]
- [14, 8]
rings:
- [9, 10, 15, 17, 18, 16]
---
structure_id: 30
number_of_atoms: 22
double_bonds:
- [1, 2]
- [6, 5]
- [20, 19]
- [21, 22]
- [3, 4]
- [7, 8]
- [11, 12]
- [13, 14]
- [9, 16]
- [10, 15]
- [17, 18]
---
structure_id: 31
number_of_atoms: 22
double_bonds:
- [1, 2]
- [6, 5]
- [20, 19]
- [21, 22]
- [3, 4]
- [7, 8]
- [11, 12]
- [13, 14]
- [9, 10]
- [15, 17]
- [16, 18]
---
structure_id: 32
number_of_atoms: 22
double_bonds:
- [1, 2]
- [6, 5]
- [20, 19]
- [21, 22]
- [3, 4]
- [7, 8]
- [11, 12]
- [13, 14]
rings:
- [9, 10, 15, 17, 18, 16]
---
structure_id: 33
number_of_atoms: 22
double_bonds:
- [1, 2]
- [6, 5]
- [20, 19]
- [21, 22]
- [3, 4]
- [9, 16]
- [10, 15]
- [17, 18]
rings:
- [7, 8, 14, 13, 12, 11]
---
structure_id: 34
number_of_atoms: 22
double_bonds:
- [1, 2]
- [6, 5]
- [20, 19]
- [21, 22]
- [3, 4]
- [9, 10]
- [15, 17]
- [16, 18]
rings:
- [7, 8, 14, 13, 12, 11]
---
structure_id: 35
number_of_atoms: 22
double_bonds:
- [1, 2]
- [6, 5]
- [20, 19]
- [21, 22]
- [3, 4]
rings:
- [7, 8, 14, 13, 12, 11]
- [9, 10, 15, 17, 18, 16]
---
structure_id: 36
number_of_atoms: 22
double_bonds:
- [1, 2]
- [6, 5]
- [20, 19]
- [21, 22]
- [11, 12]
- [13, 14]
- [15, 17]
- [16, 18]
rings:
- [3, 4, 7, 8, 9, 10]
---
structure_id: 37
number_of_atoms: 22
double_bonds:
- [20, 19]
- [21, 22]
- [7, 11]
- [12, 13]
- [14, 8]
- [9, 16]
- [10, 15]
- [17, 18]
rings:
- [1, 2, 3, 4, 5, 6]
---
structure_id: 38
number_of_atoms: 22
double_bonds:
- [20, 19]
- [21, 22]
- [7, 11]
- [12, 13]
- [14, 8]
- [9, 10]
- [15, 17]
- [16, 18]
rings:
- [1, 2, 3, 4, 5, 6]
---
structure_id: 39
number_of_atoms: 22
double_bonds:
- [20, 19]
- [21, 22]
- [7, 11]
- [12, 13]
- [14, 8]
rings:
- [1, 2, 3, 4, 5, 6]
- [9, 10, 15, 17, 18, 16]
---
structure_id: 40
number_of_atoms: 22
double_bonds:
- [20, 19]
- [21, 22]
- [7, 8]
- [11, 12]
- [13, 14]
- [9, 16]
- [10, 15]
- [17, 18]
rings:
- [1, 2, 3, 4, 5, 6]
---
structure_id: 41
number_of_atoms: 22
double_bonds:
- [20, 19]
- [21, 22]
- [7, 8]
- [11, 12]
- [13, 14]
- [9, 10]
- [15, 17]
- [16, 18]
rings:
- [1, 2, 3, 4, 5, 6]
---
structure_id: 42
number_of_atoms: 22
double_bonds:
- [20, 19]
- [21, 22]
- [7, 8]
- [11, 12]
- [13, 14]
rings:
- [1, 2, 3, 4, 5, 6]
- [9, 10, 15, 17, 18, 16]
---
structure_id: 43
number_of_atoms: 22
double_bonds:
- [20, 19]
- [21, 22]
- [9, 16]
- [10, 15]
- [17, 18]
rings:
- [1, 2, 3, 4, 5, 6]
- [7, 8, 14, 13, 12, 11]
---
structure_id: 44
number_of_atoms: 22
double_bonds:
- [20, 19]
- [21, 22]
- [9, 10]
- [15, 17]
- [16, 18]
rings:
- [1, 2, 3, 4, 5, 6]
- [7, 8, 14, 13, 12, 11]
---
structure_id: 45
number_of_atoms: 22
double_bonds:
- [20, 19]
- [21, 22]
rings:
- [1, 2, 3, 4, 5, 6]
- [7, 8, 14, 13, 12, 11]
- [9, 10, 15, 17, 18, 16]
