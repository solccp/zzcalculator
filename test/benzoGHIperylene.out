 14 + 21 x + 9 x^2 + 1 x^3
 total: 45
